pkg/util/arrays_test.go

```diff
@@ -123,3 +124,30 @@ func TestContainsAll(t *testing.T) {
 		})
 	}
 }
+
+func TestGenerateMD5Hash(t *testing.T) {
+	map1 := map[string]string{"key": "value"}
+	map2 := map[string]string{"key": "value"}
+	hashToCompare := GenerateMD5Hash(map1)
+	assert.NotNil(t, hashToCompare)
+
+	type args struct {
+		source map[string]string
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{"Should have the same hash as map1", args{map2}, hashToCompare},
+		{"Should have no hash", args{map[string]string{}}, ""},
+		{"Should have no hash because nil", args{nil}, ""},
```

Comment:
Please add a test case making sure that maps with different contents should have different hash.

Reply:
Done